shared/js/background/redirect.es6.js

```diff
@@ -29,8 +29,10 @@ function handleRequest (requestData) {
 
     // For main_frame requests: create a new tab instance whenever we either
     // don't have a tab instance for this tabId or this is a new requestId.
-    if (requestData.type === 'main_frame') {
-        if (!thisTab || (thisTab.requestId !== requestData.requestId)) {
+    //
+    // Safari doesn't have specific requests for main frames
+    if (requestData.type === 'main_frame' && window.chrome) {
+        if (!thisTab || thisTab.requestId !== requestData.requestId) {
```

Comment:
Note, no functional changes here. I just wanted to make it extra clear that this doesn't run in Safari.